recipes-kernel/kernel-modules/kernel-module-qca9377_2.0.bb

```diff
@@ -0,0 +1,17 @@
+require kernel-module-qcacld-lea.inc
+
+SUMMARY = "Qualcomm WiFi driver for QCA module 9377"
+
+EXTRA_OEMAKE += " \
+    CONFIG_CLD_HL_SDIO_CORE=y \
+    CONFIG_FEATURE_COEX_PTA_CONFIG_ENABLE=y \
+    CONFIG_PER_VDEV_TX_DESC_POOL=1 \
+    CONFIG_QCA_LL_TX_FLOW_CT=1 \
+    CONFIG_QCA_SUPPORT_TXRX_DRIVER_TCP_DEL_ACK=y \
+    CONFIG_WLAN_FEATURE_FILS=y \
+    CONFIG_WLAN_WAPI_MODE_11AC_DISABLE=y \
+    MODNAME=qca9377 \
+    SAP_AUTH_OFFLOAD=1 \
+"
+
+RDEPENDS_${PN} += "firmware-qca9377"
```

Comment:
The RDEPENDS for firmware-qca* and for qca-tools do not seem to have the intended effect. The resulting image has the kernel-module but is missing the firmware and the tools.

Reply:
I added a commit to work around the problem with dependencies. Can you review the pull request and consider accepting it with the workaround?